### ISSUE TITLE
Test crashes: Data too long for column 'results'

### ISSUE THREAD

**Issue by matsduf** [zonemaster/zonemaster-backend#616]
Tests results in error message "DBD::mysql::db do failed: Data too long for column 'results' at row 1 at /usr/local/share/perl/5.26.1/Zonemaster/Backend/DB/MySQL.pm line 208." 

Field "results" is of data type `BLOB`, if I get it right, is limited to 64 kB data. 

I tested .com and .arpa through RPC-API, but the tests do not complete. Testing with CLI is no problem. When I test the following command

```
zonemaster-cli --show_module --json com --level INFO > test_com.json
```
the resulting file `test_com.json` is 76 kB.

```
$ curl -sH "Content-Type: application/json" -d '{"jsonrpc": "2.0","id": 6,"method": "get_test_results","params": {"id": "fcb73b0a8f5b056a","language": "en"} }' http://127.0.0.1:5000/ | jq .
{
  "result": {
    "creation_time": "2020-05-15 20:51:24",
    "hash_id": "fcb73b0a8f5b056a",
    "id": 52,
    "results": [
      {
        "level": "CRITICAL",
        "message": "\n",
        "module": "BACKEND_TEST_AGENT"
      }
    ],
    "params": {
      "domain": "com",
      "priority": 10,
      "ipv6": true,
      "queue": 0
    }
  },
  "jsonrpc": "2.0",
  "id": 6
}
```

Found in `/var/log/zonemaster/zm-testagent.log` 

```
2020-05-15T20:51:25Z [27027] INFO - Test found: fcb73b0a8f5b056a
2020-05-15T20:51:25Z [5510] INFO - Test starting: fcb73b0a8f5b056a
2020-05-15T20:52:28Z [5510] ERROR - Test died: fcb73b0a8f5b056a: DBD::mysql::db do failed: Data too long for column 'results' at row 1 at /usr/local/share/perl/5.26.1/Zonemaster/Backend/DB/MySQL.pm line 208.
2020-05-15T20:52:28Z [27027] NOTICE - Worker process (pid 5510, testid fcb73b0a8f5b056a): Terminated with exit code 0
```

**Comment by matsduf** [comment]
For Postgres databases we do not seem to have any size limit problem. The field is of type `JSON` in the Postgres database, and the size limit of such is around 256 MB as far as I could read. Testing .com on installation with Postgres does not seem to be a problem. This issue is therefore limited to Mysql/Mariadb databases.

**Comment by matsduf** [comment]
Tests have shown that increasing the column from BLOB to MEDIUMBLOB will resolve the issue. See PR #617.

**Comment by matsduf** [comment]
Resolved by #617.
